### ISSUE TITLE
Placement of the [Create "xxx"] entry seems arbitrary when search returns matches

### ISSUE THREAD

**Issue by csindle** [yourlabs/django-autocomplete-light#728]
See attachment:

![dal](https://cloud.githubusercontent.com/assets/190719/17667341/d44d81e4-6304-11e6-82c3-d5543d23fe01.png)

The paginated list of results keeps the Create entry where it first appeared. 
Perhaps on scroll it should move to the front, end, or disappear (forcing the green plus to be used)?
(is it possible to force the green plus alone to be used when creating new entries?)

**Comment by jpic** [comment]
To not enable "Create ..." option, then don't set create_field.

http://django-autocomplete-light.readthedocs.io/en/master/tutorial.html#creation-of-new-choices-in-the-autocomplete-form

**Comment by jpic** [comment]
I don't want to have it first in line because i don't want to make it too easy for the user to select it by mistake. But if it's the last item on the last page of autocomplete results then the user might miss it.

However, i might be doing a mistake myself by not crediting the user with basic intelligence, perhaps these fears are emotional only ?

@gagarski @blueyed @luzfcb what do you think ? Should we move "Create" to be at one extreme or another from the complete result list ?

**Comment by blueyed** [comment]
Having it somewhere in the middle (at the end of the first page) seems strange indeed.
I should not put it first though, too.

If it is reachable at the end easily (end key), I'll put it there.
But wouldn't it jump/move when the next page gets loaded then?

**Comment by jpic** [comment]
It would be great if we could have it jump move to the end and stick there indeed.

**Comment by gagarski** [comment]
> If it is reachable at the end easily (end key), I'll put it there.

1. It seems like "End" key does not work with Select2
1. I do not think that pressing "End" key is an easy way. I am not quite sure, but it seems for me that not so many people actually use it
1. Scrolling down to the end of choices causes Select2 to load another portion of data from server.

**Comment by natehawkboss** [comment]
Yes, seems like this should be separate.  If it is within the list and always moves to the bottom, then you have to keep scrolling all the way through your options in order to click it.  It also doesn't make sense to have it at the top.

Any reason we can't have this outside of the ul and place this within its parent span.select2-results?
